opencti-platform/opencti-graphql/src/modules/fintelTemplate/fintelTemplate-domain.ts

```diff
@@ -34,21 +35,73 @@ export const findById = async (context: AuthContext, user: AuthUser, id: string)
   return storeLoadById(context, user, id, ENTITY_TYPE_FINTEL_TEMPLATE);
 };
 
+// check validity of variable_name of fintel template widgets
+const checkFintelTemplateWidgetsValidity = (fintelTemplateWidgets: FintelTemplateWidget[]) => {
```

Comment:
Not a blocker for this chunk but this method could be covered by backend unit tests.

Reply:
done :)